.prototools

```diff
@@ -0,0 +1,2 @@
+node = "~20"
```

Comment:
What is this file for?

Reply:
My bad. I should have left this file out of the commit. It's to specify that this repo uses node 20 for this tool https://moonrepo.dev/proto